try:
    4/0
except:
    print('error')
    pass

print(1)